config/checkstyle/checkstyle.xml

```diff
@@ -8,39 +8,39 @@
     https://oss-generic.github.io/process/codingstandards/coding-standards-java.html
 -->
 
-<module name="Checker">
```

Comment:
Changed the wrong files when using IntelliJ to change all occurrences of "name".

Reply:
fixed!